src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -17,6 +17,17 @@ import {
 } from './steps/ImageOutput/Environment';
 import ImageOutputStep from './steps/ImageOutput/ImageOutput';
 import ReviewStep from './steps/Review/ReviewStep';
+import AWSTarget, {
+  validateAWSAccountID,
+} from './steps/TargetEnvironment/AWS/AWSTarget';
+import AzureTarget, {
+  validateAzureId,
+} from './steps/TargetEnvironment/Azure/AzureTarget';
+import GCPTarget, {
+  GCPAccountTypes,
+  validateGCPData,
+} from './steps/TargetEnvironment/GCP/GCPTarget';
```

Comment:
I kinda prefer to have a sub folder for a substep, but I don't mind too much squashing everything up a folder. If you prefer it strongly that way I can change that.

Reply:
I don't think we need to have sub folders here since most of these steps are going to only be a single file. I would suggest just having `/steps/AWSTarget.tsx`